NBitcoin/TransactionBuilder.cs

```diff
@@ -1433,6 +1483,11 @@ bool IsEconomical(Coin c)
 					input.Sequence = 0;
 					ctx.NonFinalSequenceSet = true;
 				}
+				if (_Sequence != null)

```

Comment:
I'm not sure this won't collide with usage of RBF.

Reply:
You just need to set `ctx.NonFinalSequenceSet` to `true`.